diff -upNr desktop-3.7.4/src/gui/application.h desktop-3.7.4.patched/src/gui/application.h
--- desktop-3.7.4/src/gui/application.h	2023-03-09 09:57:57.000000000 +0100
+++ desktop-3.7.4.patched/src/gui/application.h	2023-03-11 16:06:06.147411392 +0100
@@ -134,7 +134,7 @@ private:
     QString _logDir;
     int _logExpire = 0;
     bool _logFlush = false;
-    bool _logDebug = true;
+    bool _logDebug = false;
     bool _userTriggeredConnect = false;
     bool _debugMode = false;
     bool _backgroundMode = false;
